portfoliostyle.css

```diff
@@ -0,0 +1,190 @@
+html {
+	font-size: 10px;
+	box-sizing: border-box;
+	scroll-behavior: smooth;
+}
+body {
+	margin: 0;
+
+}
+
+
+h1 {
+	font-size: 7rem;
+	color:  #FFFF00;
+	font-family: 'Anton', sans-serif;
+
+}
+
+h2 {
+	font-size: 4rem;
+	color: #b1fae2;
+	font-family: 'Anton', sans-serif;
+
+}
+
+h3 {
+	font-size: 2.9rem;
+	color: #FFFF00;
+	padding-bottom: 10px;
+	font-family: 'Anton', sans-serif;
+}
+
+p {
+	font-size: 2rem;
+	font-family: 'Exo 2', sans-serif;
+	color: #f7faf9;
+}
+
+
+
+nav {
+	background-color: #4393e8;
+	display: flex;
+	justify-content: center;
+	padding: 10px 0;
+	align-items: center;
+	position: fixed;
+	z-index: 10;
+	width: 100vw;
+}
+
+.nav_link {
+    color: #000000;
+    text-decoration: none;
+	padding: 30px;
+	font-size: 2.5rem;
+	font-weight: bold;
+}
+
+a:hover {
+		text-decoration: none;
+		color:#FAFAFA;
+		font-weight: bold;
+		font-size: 5rem;
+}
+
+header {
+	background-color: #4393e8;
+	display: flex;
+	width: 100vw;
+	text-align: center;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+	padding-top: 65px;
+	margin: 0px;
+	
+}
+
+#coder {
+	position: relative;
+	padding-top: 15px;
+	width: 30%;
```

Comment:
instead of 30% i would use max-width (similar to what we discussed about other stuff)